README.md

```diff
@@ -1 +1,46 @@
-##Network Service
+# Network Service 
+
+Network layer for creating different set of network requests like GET, POST, PUT, DELETE 
+⚠️ The network layer is under active development.
+
+## Current dev environment 
+
+| Dev environment| Ref link |
+| --- | -- |
+| ![macOS](https://img.shields.io/badge/macOS-13.3+-blue) | | 
+| ![Xcode](https://img.shields.io/badge/Xcode-14.3-red) | |
+| ![Swift](https://img.shields.io/badge/Swift-5.8-orange) | |
+| ![SDKs for iOS](https://img.shields.io/badge/SDKs%20for%20iOS-15%2B-lightgrey) | |
+| ![SwiftFormat](https://img.shields.io/badge/SwiftFormat-0.51.7-yellow) | [Link](https://github.com/nicklockwood/SwiftFormat) |
+| ![SwiftLint](https://img.shields.io/badge/SwiftLint-0.51.0-green)| [Link](https://github.com/realm/SwiftLint) |
+
+## Table of contents
+* [General info](#general-info)
+* [Installation](#installation)
+* [Authors](#authors)
+
+## General info
+
+The repo contains the source code of the NetworkService.
+
+## Installation
+
+**Pre-requisites**: _installed Xcode with the version specified under the [current dev environment](#current-dev-environment)._
+
+1.
+
+## Authors
+
+You can contact the authors by the following GitHub's usernames:
+
+- Nadia Shpakouskaya: [@NadzeyaShpakouskaya](https://github.com/NadzeyaShpakouskaya)
+- Alex Zavadskiy: [@iBYarchitect](https://github.com/iBYarchitect)
+- Julya Kmita: [@julyakmita](https://github.com/julyakmita)
+- Marina Tihonovich: [@marinkman](https://github.com/marinkman)
+- Yuliya Nastiushkina: [@YuliaNastiushkina](https://github.com/YuliaNastiushkina)
+- Ivan Kirpichonok: [@kirpichonok](https://github.com/kirpichonok)     
+- Alex Levitski: [@levvaa](https://github.com/levvaa)
+
+## License
+
+The source code for the site is licensed under the MIT license, which you can find in the LICENSE file.
```

Comment:
Just give the link to the `LICENSE` file, DRY :)

Reply:
fixed

Reply:
What was fixed. You have put the link to the MIT license, not your LICENSE file.

Reply:
changed

Reply:
I added a link to the license file, but it is in the PR

Reply:
See the description of the PR to find a link to the `LICENSE` file, in the readme there is a link to the `LICENSE` file in the main